### PR TITLE
fix(console): respect `node.title` in the map view

### LINKED ISSUE #6457: Regression: `node.title` is not respected in Wing Console
### I tried this:

```js
bring cloud;

let x = new cloud.Queue();
nodeof(x).title = "hello";
```

### This happened:

image

The tree shows the right title:

image

### I expected this:

_No response_

### Is there a workaround?

_No response_

### Anything else?

_No response_

### Wing Version

_No response_

### Node.js Version

_No response_

### Platform(s)

_No response_

### Community Notes

* Please vote by adding a 👍 reaction to the issue to help us prioritize.
* If you are interested to work on this issue, please leave a comment.

### PR DESCRIPTION
**Pull request by skyrpex**
Update the display of node names in the MapView component to show the title if available, falling back to the id if not.

Fixes #6457.

### CODE REVIEW

**Review by eladb — APPROVED** [review]
Is there a way to test this?

**Comment by skyrpex** [comment]
> Is there a way to test this?

We have to revisit the test suite and then we could start adding the tests back. But right now they are just being skipped.

**Comment by eladb** [comment]
Bummer

**Comment by ainvoner** [comment]
@skyrpex just making sure you tested:
display.title, "as", and ID 
for reference:
https://github.com/winglang/wing/pull/6211
https://github.com/winglang/wing/issues/5994

File: apps/wing-console/console/ui/src/features/map-view.tsx [hunk 1]
```diff
@@ -682,7 +682,10 @@ export const MapView = memo(
         return (
           <ConstructNode
             id={props.constructTreeNode.path}
-            name={props.constructTreeNode.id}
+            name={
```
Comment (ainvoner) [inline apps/wing-console/console/ui/src/features/map-view.tsx:685]: @skyrpex just making sure you tested:
display.title, "as", and ID 
for reference:
https://github.com/winglang/wing/pull/6211
https://github.com/winglang/wing/issues/5994

**Review by ainvoner — CHANGES_REQUESTED** [review]

**Review by ainvoner — APPROVED** [review]

**Pull request merged**